### ISSUE TITLE
Call Of Duty: Black Ops will not start: Acer Nitro 5; Ubuntu 20.04 LTS; GTX 1050

### ISSUE THREAD

**Issue by tptignor** [ValveSoftware/Proton#4633]
# Compatibility Report
- Name of the game with compatibility issues: Call Of Duty: Black Ops
- Steam AppID of the game: 42700

## System Information
- GPU:  GeForce GTX 1050
- Driver/LLVM version:  nvidia-driver-460
- Kernel version:  5.8.0-41-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/tptignor/b87d4a1685a55e6e4c962e9a82276881
- Proton version:experimental-5.13-20210217

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [ x] that I have checked whether there are updates for my system available.

[steam-42700.log](https://github.com/ValveSoftware/Proton/files/5999382/steam-42700.log)

## Symptoms 

The game fails to start. There is a cryptic exception in the log.

## Reproduction

Double click on Play in the Steam window.

**Comment by kisak-valve** [comment]
Hello @tptignor, we're using one issue report per unofficially supported game title, so I've gone ahead and transferred this issue report to https://github.com/ValveSoftware/Proton/issues/790#issuecomment-780945819.

**Comment by tptignor** [comment]
Hello kisak-valve,
I see issue #790 is marked "No one assigned," and I'm getting deja vu. You did the same thing with the last issue I opened (#4602) and I never heard from anyone. Is someone going to work on #790?
Is there actually a person here? I think I got these three emails within a couple minutes of opening the new issue.
Tom    :-/ 

    On Wednesday, February 17, 2021, 07:30:40 PM EST, kisak-valve <notifications@github.com> wrote:  
 
 


Hello @tptignor, we're using one issue report per unofficially supported game title, so I've gone ahead and transferred this issue report to #790 (comment).
